scrapy_zyte_api/_poet_item_pipelines.py

```diff
@@ -0,0 +1,88 @@
+from logging import getLogger
+from typing import Any, Set, Type
+
+from itemadapter import ItemAdapter
+from scrapy import Spider
+from scrapy.crawler import Crawler
+from scrapy.exceptions import NotConfigured
+from scrapy.utils.misc import load_object
+from scrapy_poet import InjectionMiddleware
+from web_poet.fields import get_fields_dict
+from web_poet.utils import get_fq_class_name
+from zyte_common_items.fields import is_auto_field
+
+logger = getLogger(__name__)
+
+
+class ScrapyZyteAPIPoetItemPipeline:
```

Comment:
Is the idea to bake all scrapy-zyte-api related features to this pipeline, if needed? Or is this pipeline for computing the stats for overridden fields? If it's the latter, should it be renamed, to reflect it?